documentation/catalog/guides/quickstart.md

```diff
@@ -21,7 +21,7 @@ project, see [DAGs.md](../reference/DAGs.md).
 See each provider API script's notes in their respective [handbook][ov-handbook]
```

Comment:
It looks like the changes to this file were made automatically, should they be reverted? 😅